test/unittests/validation_test.cpp

```diff
@@ -13,6 +13,15 @@
 using namespace fizzy;
 using namespace fizzy::test;
 
+TEST(validation, imported_function_type_invalid_arity)
+{
+    /* wat2wasm
```

Comment:
wabt doesn't throw an error unless function with this type is defined

Reply:
Can you add this to our list of "to reach out"? Or just ask the question on the wabt repo? It may be we are using an old wabt and it was fixed since, or perhaps they are more committed to the multi-return-value change (I believe that is implemented by wabt).

Reply:
I tried it with recent wabt 1.0.19, and it appears to be fixed: multi-value is enabled by default there, but with `--disable-multi-value` this correctly throws an error.

I think I'll add `--no-check` here

Reply:
1.0.14 is the last one that has multi-value disabled by default, and it has this issue fixed also.